content/en/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -168,6 +168,8 @@ in the section [Interlude: built-in node labels](#interlude-built-in-node-labels
 processing which can slow down scheduling in large clusters significantly. We do
 not recommend using them in clusters larger than several hundred nodes.
 
+**Note:** If the specified `topologyKey` is missing on the nodes or misspelled in the pod manifest, this can lead to unintended behaviour. The scheduler will still schedule the pod(s) but ignore any `podAntiAffinity` rule specified. This behaviour only applies to inter-pod **anti-affinity**.
```

Comment:
I would like us to rephrase this. Rather than promise particular behavior when labels are missing, let's state that this feature expects all nodes to be labeled (i.e. they all have a label with the same key as the topologyKey). If some or all nodes are missing the label, it can lead to unintended behavior.

Reply:
Rephrased, PTAL @misterikkit